packages/core/README.md

```diff
@@ -5,9 +5,9 @@
 
 ## Usage
 
-### Initializing:
+### Initializing and registering the extension
 
-Initializing the SDK should be done in native code, documentation on how to initialize the SDK can be found [here](https://aep-sdks.gitbook.io/docs/getting-started/get-the-sdk#2-add-initialization-code). The linked documentation initializes the User Profile extension which is not required, however, if you are interested in the User Profile extension for React Native you can find it [here](https://www.npmjs.com/package/@adobe/react-native-aepuserprofile).
+Initializing the SDK should be done in native code, documentation on how to initialize the SDK can be found [here](../../README.md#initializing).
```

Comment:
Will this readme be the main page in npm? If so let use the absolute link of the Beta home readme page? https://github.com/adobe/aepsdk-react-native

Reply:
Good catch!